### ISSUE TITLE
Allow starring issues

### ISSUE THREAD

**Issue by dilijev** [dear-github/dear-github#208]
I don't necessarily want to subscribe for updates but I might want to come back to a particular issue. I'd like to be able to star it so I can get back to it later.

Additionally, subscribing to an issue doesn't even solve the problem of getting back to an issue. If there's never an update, I'll never see the issue again.

(Also, add the option to make stars on repositories and issues private. Right now, everyone can see what I've starred, whether I want them to or not.)

**Comment by strugee** [comment]
I _do_ see the value in this being built in to GitHub but I also feel obligated to point out that this sounds a lot like bookmarks, right down to the proposed star UI :P

**Comment by dilijev** [comment]
Yeah, I'm using bookmarks for this now and I find it to be in inadequate.

You could also argue that I don't need to star emails or documents in those services because bookmarks are good enough.

My feeling is that it's connected closely enough with the service that the service should have a UI for it, and it should follow my account, no matter what device or browser I'm using or whether I've synced my bookmarks to that device or browser.

**Comment by strugee** [comment]
As I said, I understand why you're proposing this. I just find it funny :)

**Comment by dilijev** [comment]
¯\_(ツ)_/¯
